application/client/src/app/ui/views/toolbar/search/input/component.ts

```diff
@@ -61,17 +73,21 @@ export class ViewSearchInput
             this.session.search
                 .state()
                 .subjects.get()
-                .start.subscribe(() => {
+                .start.subscribe((filters: IFilter[]) => {
                     this.progress.start();
+                    if (filters.length === 1) {
+                        this.active = new ActiveSearch(this.session.search, filters[0]);
+                    }
```

Comment:
why we need it?

Reply:
Do you mean the check or the whole thing?
The check is because we might have more than one element in the array. The assignment is because we create the active search there